### PR TITLE
Bugfix the anti-ninth

### LINKED ISSUE #1868: Plasma amount numbers don't fit on the Alien UI
saying- hello installation one 2013-11-21 224512

I noticed this while testing another bug related to runtimes with clientless aliens.

### LINKED ISSUE #2878: OneHuman is impossible to remove
##### Revision that this bug was encountered on (REQUIRED!! Type 'show-server-revision' ingame to find out - note that 'current revision' isn't helpful!): #####
89d684a244916e0dec408642af1706b0e0826a5e

##### Which server were you playing on? (REQUIRED!! e.g. TG-Sibyl1, TG-Sibyl2, Badgerstation, Facepunch, Pinkstation, /VG/... etc.) #####
Urist McStation

###### What did you expect to happen? ######
To be able to purge or otherwise overwrite a OneHuman upload on an AI.

###### What happened instead? ######
Squat. Purging, giving it a new core module, and resets all do nothing to remove OneHuman.

###### Why is this bad/What are the consequences? ######
An unsuspecting crew resetting/purging/re-Asimoving the OneHuman'd AI only to have it strike again.

##### Steps to reproduce the problem (remember to be specific! Those little details can save us hours of work!): #####
1. Upload OneHuman
2. Purge/reset/Asimov AI
3. Check laws with the admin command, note how the OneHuman law is still there
4. Cry
5. Upload lawset that doesn't care about humanity/kill AI/beg for your robot overlord's mercy

##### Possibly related stuff (which gamemode was it? What were you doing at the time? Was anything else out of the ordinary happening?): #####
Nothing.

##### Anything else? #####
I can only assume this is a consequence of OneHuman uploading into the zeroeth slot, like the traitor AI law, making it impossible to remove. This is something fairly easy to get around, namely by uploading PALADIN, Corporate, or Tyrant, but it's still pretty weird.

### LINKED ISSUE #4239: hivebot beacon is still a mob
Remember that big gray circle that spawn mobs? It got changed long time ago to a mob and nobody ever fixed it or even reported it. 
Now the beacon goes and try to bite people, it dies quickly because the hivebots that he summons kill him on their stupid friendly-fire.

Their path:
/mob/living/simple_animal/hostile/hivebot/tele

This thing used to be an event it was kinda cool.

### LINKED ISSUE #4400: ProtectStation needs better defined wording
SubtleGraces on the forums:
"ProtectStation is absolutely terrible. Had a bit of a discussion about it in ASAY after an AI had someone killed over a floor tile being pulled up.
Can we get rid of it, or better yet, have better wording for the law so nobody can conclude that simple maintenance and deconstruction deserves death?

The wording of the law is absolutely atrocious and nobody seems to have reached a consensus on what "damage" means."

### LINKED ISSUE #5148: Paper bins have no click delay
Kind of irritating. Self explanatory. You can mass-empty paper bins in hotkey mode by spamming click and pressing Q to drop the papers.

### LINKED ISSUE #5426: You can only shift-click examine your borg modules when you're not in a closet/recharger/tube
So if, say, you're a janiborg and want to know how many lights you have while recharging, you have to exit the recharger to check.
Examine code!

### PR DESCRIPTION
**Pull request by Jordie0608**
Fixes #1868, unrounded counter on UI elements
Fixes #4239, hivebot beacons being weird as a mob
Fixes #5148, paper bins having no click delay
Fixes #5426, unable to examine inventory when inside objects

Hivebot beacon now waits 1-60 seconds after spawning to activate rather than having a 2% chance every tick

More to come

### CODE REVIEW

**Comment by Dorsisdwarf** [comment]
This is a feature because 4400 is a feature request.

File: code/game/objects/structures/hivebot.dm [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	while(bot_amt > 0)
+		bot_amt--
+		switch(bot_type)
+			if("norm")
+				new /mob/living/simple_animal/hostile/hivebot(get_turf(src))
+			if("range")
+				new /mob/living/simple_animal/hostile/hivebot/range(get_turf(src))
+			if("rapid")
+				new /mob/living/simple_animal/hostile/hivebot/rapid(get_turf(src))
+	spawn(100)
```
Comment (Miauw62) [inline code/game/objects/structures/hivebot.dm]: This spawn can be replaced by a sleep.

**Comment by GunHog** [comment]
Removal of things like that should not be coupled with fixes. Feature removals should be their own PR, so they may be judged separately without delaying bugfixes, completely unrelated ones at that.

This also violates the 'atomic' PR rule, not that anyone respects that one, anyway.

**Comment by Jordie0608** [comment]
[It](https://github.com/tgstation/-tg-station/pull/9356) [closes](https://github.com/tgstation/-tg-station/pull/9587) [an](https://github.com/tgstation/-tg-station/pull/9358) [Issue](https://github.com/tgstation/-tg-station/pull/9409) [Report](https://github.com/tgstation/-tg-station/pull/9476)

**Comment by CorruptComputer** [comment]
Atomic PR rule?

**Comment by Jordie0608** [comment]
https://tgstation13.org/wiki/Getting_Your_Pull_Accepted#4._Pull_requests_must_be_atomic

**Comment by tgstation-server** [comment]
2878 is a balance concern and ideally shouldn't be rushed along in a mass
bugfix pr

**Comment by Jordie0608** [comment]
Fiiiiiiine

**Comment by Jordie0608** [comment]
They're both gone now, I cbb to correct the line endings

**Comment by optimumtact** [comment]
Fucking correct the line endings REEEEEEEEEEEEEEEEEEEEEE

**Comment by Jordie0608** [comment]
Too slow scrub, already fixed

**Pull request merged**